### PR TITLE
Copy in code from django-bootstrap, set up package

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: seaworthy/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from .helper import DockerHelper
```
Comment (jerith) [inline seaworthy/__init__.py]: Maybe this should be `dockerhelper` rather than `helper`?

File: seaworthy/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from testtools.assertions import assert_that
+from testtools.matchers import Equals
+
+from ..utils import resource_name
```
Comment (jerith) [inline seaworthy/tests/test_utils.py]: Can we make this an absolute import rather?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    packages=find_packages(),
+    install_requires=requirements,
+    tests_require=test_requirements,
+    extras_require={
+        'test': test_requirements,
+        'pep8test': [
+            'flake8',
+            'flake8-import-order',
+            'pep8-naming',
+        ],
```
Comment (jerith) [inline setup.py]: Why are these deps inline when the others are split out?

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+[testenv]
+extras =
+    test
+deps =
+    coverage
+commands =
+    pip list
+    coverage run -m pytest {posargs}
+    coverage report -m
```
Comment (jerith) [inline tox.ini]: Should we maybe use `pytest-cov` for this?

File: seaworthy/tests/test_utils.py (hunk 2 above)
Comment (JayH5) [inline seaworthy/tests/test_utils.py]: The `flake8-import-order` thing was complaining with an absolute import for some reason and I was too lazy to figure out why :/

File: setup.py (hunk 3 above)
Comment (JayH5) [inline setup.py]: No real reason. They could all be inline. I guess the others have a greater chance of changing and also have a chance of having Python version-specific deps being added.

File: tox.ini (hunk 4 above)
Comment (JayH5) [inline tox.ini]: We can, I'm just still biased towards using `coverage` directly 😛 but don't have a strong argument.

File: seaworthy/tests/test_utils.py (hunk 2 above)
Comment (JayH5) [inline seaworthy/tests/test_utils.py]: Figured it out

File: seaworthy/utils.py [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+    raise RuntimeError('Log pattern {!r} not found in logs.{}'.format(
+        pattern, _last_few_log_lines(container)))
+
+
+def output_lines(raw_output, encoding='utf-8'):
+    return raw_output.decode(encoding).splitlines()
+
+
+@attr.s
```
Comment (JayH5) [inline seaworthy/utils.py]: I'm thinking this process stuff could be in its own module.

Comment (jerith) [inline seaworthy/utils.py]: Seems reasonable.

File: README.rst [hunk 6]
```diff
@@ -10,11 +10,16 @@ seaworthy
 
 Test harness for Docker container images
 
+Seaworthy has similar goals to `TestContainers`_, although it is written in
+Python and is based on our experience with testing Docker images.
```
Comment (jerith) [inline README.rst]: Based on the TestContainers docs, this isn't entirely true. While there's a lot of overlap, TestContainers seems to focus on providing test dependencies in containers whereas we focus specifically on testing that containers work.

**Pull request merged**